src/hooks/screen/useChatScreenHook.ts

```diff
@@ -108,7 +137,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       if (e?.response?.data?.status === 'Channel is blocked') return;
 
       setTimeout(() => {
-        sendChat(message, iteration + 1, currentChatSchema).catch((sendChatError) => {
+        sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function is now being called with the `attachments` parameter. This change looks good as long as the `sendChat` function is updated to handle this new argument.
```diff
+         sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->